pkg/pool-utils/contracts/PoolInfo.sol

```diff
@@ -36,12 +36,12 @@ contract PoolInfo is IPoolInfo {
     }
 
     /// @inheritdoc IPoolInfo
-    function getCurrentLiveBalances() external view returns (uint256[] memory balancesLiveScaled18) {
+    function getCurrentLiveBalances() public view returns (uint256[] memory balancesLiveScaled18) {
```

Comment:
Since they're one-liners, could leave these external and just inline it in the getter, like the token rates. Same with `getStaticSwapFeePercentage`